puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
+    wrappers:
+      - timeout:
+          type: absolute
+          timeout: {timeout}
```

Comment:
https://github.com/domcleal/foreman-infra/commit/028914b8e4df27728dac4ebe20fb76650d102fd0 seems to work for me.